lib/msal-common/src/response/AuthorizationCodePayload.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Response returned after processing the code response query string or fragment.
+ */
+export type AuthorizationCodePayload = {
+    code: string,
+    cloud_instance_name?: string;
+    cloud_instance_host_name?: string;
+    cloud_graph_host_name?: string;
+    msgraph_host?: string;
+    state?: string;
```

Comment:
state and nonce should be required, no?

Reply:
No because these are set by the platform library. They are not required in msal-node.

Reply:
Ah right, @sameerag  was going to add them to node and will change this then